server.js

```diff
@@ -10,18 +10,16 @@ const injector = require('./injector.js')
 
 const repository = '/repository/'
```

Comment:
Changing this to:
> const repository = './repository/'

will fix the problem with the creation of a repository folder